src/Murder.Editor/Assets/SpriteEventDataManagerAsset.cs

```diff
@@ -6,7 +6,6 @@
 
 namespace Murder.Editor.Assets;
 
-[RuntimeOnly]
 internal class SpriteEventDataManagerAsset : GameAsset
```

Comment:
actually! for this, I was using it as a cheat hahah. this is a "runtime" asset, which means that I create this to manage assets internally but I never want to explicit this to the user and it's just an implementation detail:
https://github.com/isadorasophia/murder/blob/b349b8e79fcefe646ff3c470afdd3f96c39fe81f/src/Murder.Editor/EditorScene_Selector.cs#L33

this is the case for this asset, which I use to track modifications to aseprite files when generating sprite assets. do you think `RuntimeOnly `is confusing here and we should create another attribute for these types of assets, or keep it `RuntimeOnly`?

Reply:
I think we can either create a more explicit attribute (`[InternalAsset]` or reuse `[HideInEditor]`) or we can change the documentation of `RuntimeOnly`. I think the former is a cleaner solution, since `RuntimeOnly` is meant to be exposed to end users while this is pretty engine specific

Reply:
sounds good!